src/app/plugins/twitter.plugin.ts

```diff
@@ -0,0 +1,99 @@
+import { MessageEmbed, TextChannel } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+import { TwitterTimelineResponse, TwitterService } from '../../services/twitter.service'
+
+export class TwitterPlugin extends Plugin {
+
+    public name = 'twitter';
+    public description = 'Gets the latest twitter timelines from UCF accounts';
+    public permission = ChannelType.Public;
+    public usage = 'twitter <UCF account>';
+
+    private static twitterIconURL = 'https://images-ext-1.discordapp.net/external/bXJWV2Y_F3XSra_kEqIYXAAsI3m1meckfLhYuWzxIfI/https/abs.twimg.com/icons/apple-touch-icon-192x192.png';
+    private static accounts: Record<string, string> = {
+        ucf: '18999501',
+        knights: '21306514',
+        football: '30282826',
+        knighthacks: '3122136832',
+        cecs: '2292877801',
+    }
+
+    // The twitter API returns a min of 5 tweets, but thats a bit much for our bot, so we'll do 3 instead.
+    private maxSize = 3;
+    private twitter = new TwitterService();
+    
+    public constructor(public container: IContainer) {
+        super();
+        this.twitter = container.twitterService;
+    }
+
+    public async execute(message: IMessage, args: string[]) {
+        const [param] = args;
+
+        let accountId;
+
+        // Default to UCF account if no args provided.
+        if (!param) {
+            accountId = TwitterPlugin.accounts.ucf;
+        } else {
+            accountId = TwitterPlugin.accounts[param.toLowerCase()];
+        }
+
+        // Show possible options if invalid account was specified.
+        if (!accountId) {
+            let options = '\n';
+            Object.keys(TwitterPlugin.accounts).forEach(key => options += `🔸 ${key}\n`);
+            message.react('❌')
+            message.reply(`Invalid UCF Twitter Account \'${param}\', possible options are:${options}`);
+            return;
+        }
+
+
+        message.react('👍');
+        message.reply('Sure thing! Getting latest tweets!');
+
+        // Fetch respective tweets.
+        const response = await this.twitter.getLatestTweets(accountId, this.maxSize);
+        const embeds = await this._createEmbeds(response, accountId);
+
+
+        // The reason a webhook is used here is because traditional bot messages don't allow you
+        // to send multiple embeds at once, with a webhook you can send 10 at a time.
+        const webhook = await (message.channel as TextChannel).createWebhook('UCF Twitter', {
+            avatar: 'https://about.twitter.com/content/dam/about-twitter/en/brand-toolkit/brand-download-img-1.jpg.twimg.1920.jpg'
+        });
+        webhook.send({ embeds });
```

Comment:
await

Reply:
This is the last statement in the block and the promised value isn't used, the `send()` message will execute async but we don't care about the result, so prefixing with `await` here wouldn't make a difference.